src/test/user.spec.js

```diff
@@ -890,3 +891,111 @@ describe('GET /api/v1/users/followings', () => {
     expect(res.body.metadata.totalItems).to.equal(numberOfFollowing.count);
   });
 });
+
+describe(' POST /api/v1/auth/reset-password', async () => {
+  const newUser = await createUser();
+  const user = await BaseRepository.findAll(db.User, {
+    id: newUser.email
+  });
+  it('it should send password reset link to the email provided if it is associated with a registered email', done => {
+    const token = helper.jwtSigner(600, {
+      email: newUser.email
+    });
+    chai
+      .request(app)
+      .post('/api/v1/auth/reset-password')
+      .set('x-access-token', token)
+      .send({
+        email: 'olorunwalawrence5@gmail.com'
+      })
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body.message).to.be.equal(
+          'A password reset link would be sent to the email provided if it is associated with a registered email'
+        );
+      });
+
+    done();
+  });
+
+  it('should return invalid email credential', done => {
+    const token = helper.jwtSigner(600, {
```

Comment:
The argument passed to the` jwtSigner` doesn't follow the function signature. The payload should come first while the expire time comes as the last argument.